src/widget.ts

```diff
@@ -85,6 +87,7 @@ abstract class ReteIOModel extends DOMWidgetModel {
 
   key: string;
   title: string;
+  multi_connection:boolean;
```

Comment:
```suggestion
  multi_connection: boolean;
```

adding a space, for linting purposes